front/src/components/LanguageChanger/LanguageChanger.test.js

```diff
@@ -0,0 +1,44 @@
+import React from 'react';
+import { render } from '@testing-library/react';
+import { LanguageChanger } from './index';
+import { Logo } from '../Logo';
+import '../../i18n';
+import '@testing-library/jest-dom/extend-expect';
+
+describe('Language changer', () => {
+  test('should be default language', () => {
+    const { container } = render(
+      <LanguageChanger defaultLanguage={'en'} />
+    );
+    expect(container).toMatchSnapshot();
+  });
+  test('should be list of languages', () => {
+    const { container, getByText } = render(
+      <LanguageChanger defaultLanguage={'en'} />
+    );
+    const button = container.querySelector('.fa-chevron-down')
+      .parentElement;
```

Comment:
Several things bother me in this line of code:
1. The variable name is too generic. Can we use a better name to emphasize the semantic meaning of that button?
2. The selector has dependency on a class name and layout (`.parentElement`). What are the pros and cons of such an approach in your opinion? Can we somehow decouple those things?